generator/templates/rust/crate/src/lib.rs.mustache

```diff
@@ -106,6 +106,10 @@ impl Client {
     }
     {{/Services}}
 }
+
+#[derive(serde::Serialize)]
+#[allow(dead_code)]
+struct NoBody {}
```

Comment:
If I was clever I would suggest a way to do this with the unit type...

Reply:
Yeah, I could not figure something like that out...